doc/classes/Geometry3D.xml

```diff
@@ -81,6 +81,33 @@
 			<param index="3" name="c" type="Vector3" />
 			<description>
 				Returns a [Vector3] containing weights based on how close a 3D position ([param point]) is to a triangle's different vertices ([param a], [param b] and [param c]). This is useful for interpolating between the data of different vertices in a triangle. One example use case is using this to smoothly rotate over a mesh instead of relying solely on face normals.
+				[param a], [param b] and [param c] must form a valid triangle and [param point] must be coplanar with the [param a][param b][param c] triangle. Returns an empty [Vector3] if [param a][param b][param c] does not form a valid triangle. Returns invalid weights if [param point] is not coplanar with the [param a][param b][param c] triangle.
+				[codeblock]
+				var a = Vector3(1, 1, 0)
+				var b = Vector3(1, 1, 1)
+				var c = Vector3(2, 1, 0)
+				var point = Vector3(1.5, 1, 0.5)
+				var w = get_triangle_barycentric_coords(point, a, b, c)
+				# w is Vector3(0, 0.5, 0.5)
+				# w contains valid weights
```

Comment:
WDYT about something like:
```suggestion
				var point = Vector3(1.5, 1, 0.5) # Coplanar with abc triangle (valid input)
				var w = get_triangle_barycentric_coords(point, a, b, c) # w is Vector3(0, 0.5, 0.5)
				point.is_equal_approx(w.x * a + w.y * b + w.z * c) # True, w contains valid weights
```
?
(if it's good then applicable to the other examples)

Reply:
Thanks for your suggestions!

When I have time, I'll add them.

Reply:
Yes, I like this.